### DISCUSSION TITLE
登录验证失败。机场和pcl自带的处理方法无法解决

### DISCUSSION THREAD

**Discussion by ZiChuan7** [Meloong-Git/PCL#3722] [提问]
![dcd34878ab26f71fd95830c4629c3630](https://github.com/Hex-Dragon/PCL2/assets/167798869/18824f71-dc80-4617-95d8-22779e1a7e4e)

**Comment by wuliaodexiaoluo** [comment]
检查你的账户是否配置了强制登录禁令

**Reply by MoYuan-CN** [reply]
Author 开了 #3723 来着...
